src/frontend/src/lib/components/send/Send.svelte

```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	import { modalStore } from '$lib/stores/modal.store';
+	import { modalSend } from '$lib/derived/modal.derived';
+	import SendButton from '$lib/components/send/SendButton.svelte';
+	import SendModal from '$lib/components/send/SendModal.svelte';
+
+	const modalId = Symbol();
+</script>
+
+<SendButton on:click={() => modalStore.openSend(modalId)} />
+
+{#if $modalSend && $modalStore?.data === modalId}
```

Comment:
Unrelated to this PR, thinking out loud, maybe we can create a generic component?

```
<script>
   export let open = false;

   const modalId = Symbol();
<script>

{#if open && $modalStore?.data === modalId}
  <slot />
{/if}

// Used

<ModalGuard open={$modalSend} />
   <SendModal on:nnsClose />
</ModalGuard>
```

Maybe the modal can even be passed as a component?

```
<ModalSomething open={$modalSend} cmp={SendModal} />
```

Again, just thinking out loud. Sharing if interesting.

Reply:
Done with:

- PR #1978
- PR #1979